asio/include/asio/detail/config.hpp

```diff
@@ -617,9 +617,9 @@
 #  if (__cplusplus >= 201703)
 #   if defined(__clang__)
 #    if defined(ASIO_HAS_CLANG_LIBCXX)
-#     if (_LIBCPP_STD_VER > 14) && defined(_LIBCPP_HAS_ALIGNED_ALLOC)
+#     if (_LIBCPP_STD_VER > 14) && defined(_LIBCPP_HAS_ALIGNED_ALLOC) && !defined(_LIBCPP_MSVCRT_LIKE)
```

Comment:
@chriskohlhoff should I close this PR? I noticed you just took this change https://github.com/chriskohlhoff/asio/commit/92aacba13a222c6d29f090a929b13695a6a893f2 instead of working with me on this one...